src/uid2CallbackManager.ts

```diff
@@ -20,22 +20,26 @@ export class Uid2CallbackManager {
   private _getIdentity: () => Uid2Identity | null | undefined;
   private _logger: Logger;
   private _sdk: UID2SdkBase;
+  private _productName: string;
   constructor(
     sdk: UID2SdkBase,
+    productName: string,
     getIdentity: () => Uid2Identity | null | undefined,
     logger: Logger
   ) {
+    this._productName = productName;
     this._logger = logger;
     this._getIdentity = getIdentity;
     this._sdk = sdk;
     this._sdk.callbacks.push = this.callbackPushInterceptor.bind(this);
   }
 
-  private static _sentSdkLoaded = false; //TODO: This needs to be fixed for EUID!
+  private static _sentSdkLoaded: Record<string, boolean> = {}; //TODO: This needs to be fixed for EUID!
```

Comment:
Does this comment still apply?

Reply:
Good catch - no, this fixes it for EUID :) I'll update it.